modules/kubernetes_node_pool/aks/0.1/facets.yaml

```
@@ -32,46 +32,51 @@ spec:
-    - disk_size
-    - taints
-    - labels
+  - instance_type
+  - min_node_count
+  - max_node_count
+  - disk_size
+  - taints
+  - labels
 sample:
-  $schema: https://facets-cloud.github.io/facets-schemas/schemas/kubernetes_node_pool/kubernetes_node_pool.schema.json
+  $schema: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)